### DISCUSSION TITLE
Adding Multisort on Custom Sorted Column

### DISCUSSION THREAD

**Discussion by at-webdev** [TanStack/table#2100] [General]
Hello All, 
I have a custom sorted column, and I would like to know how to make it support for useSortby's multi sort feature.

when clicked on Column Name , followed with Column Items, it multi sorts correctly.
But when clicked on Column Items and followed with any other column click, it doesnt support multisort.
Is there any multisort option that I am missing to add on my customSort function.

Table:
```
  const { getTableProps, headerGroups, rows, prepareRow } = useTable(
    {
      columns,
      data,
      isMultiSortEvent: () => {
        return true;
      },
    },
    useSortBy
  );
```
Custom Sort based on Array Length
```
const customSort = useCallback((rowA, rowB, columnId)=>{
return rowA.values[columnId].length > rowB.values[columnId].length ? 1 : -1
})
```

Column:
```
const columns = useMemo(()=>[
{
header:"Name",
accessor:"name",
sortType:"basic"
},
{
header:"Age",
accessor:"age"
},
{
header:"Items",
accessor:"items",
sortType:customSort,
Cell:({cell:{val}})=>(
val.map((value:string)=>(
<span>value</span>
)
)
}
],[customSort]);
```

Data:
```
const data = useMemo(()=>[
{
name:"Sam",
age:"40",
items:["A","B","C"]
},
{
name:"Sam",
age:"35,
items:["A"]
},
{
name:"Mike",
age:12,
items:["B","C"]}
],[]);
```

**Comment by romaloshak** [comment]
thank u man, u help me to solve custom multi sort without shift key
